data_steward/tools/create_combined_backup_dataset.py

```diff
@@ -0,0 +1,552 @@
+"""
```

Comment:
@ksdkalluri I obviously missed something during the first review.  Why are we creating this new module when `tools/combine_ehr_rdr.py` already exists and has been combining the data?  Is this intended to replace that module, work with that module, or has that one been forgotten for some reason?